packages/gdc/src/GDCFeatureDrawerWidget/index.js

```diff
@@ -1,21 +1,25 @@
 import { ConfigurationSchema } from '@gmod/jbrowse-core/configuration'
 import { ElementId } from '@gmod/jbrowse-core/util/types/mst'
-import { types } from 'mobx-state-tree'
 
-export const configSchema = ConfigurationSchema('GDCFeatureDrawerWidget', {})
-export const stateModel = types
-  .model('GDCFeatureDrawerWidget', {
-    id: ElementId,
-    type: types.literal('GDCFeatureDrawerWidget'),
-    featureData: types.frozen({}),
-  })
-  .actions(self => ({
-    setFeatureData(data) {
-      self.featureData = data
-    },
-    clearFeatureData() {
-      self.featureData = {}
-    },
-  }))
+export default jbrowse => {
+  const { types } = jbrowse.jbrequire('mobx-state-tree')
+  const configSchema = ConfigurationSchema('GDCFeatureDrawerWidget', {})
```

Comment:
should configuration schema be a re-export?

Reply:
yeah probably should. i think further work on this will go into the GDC plugin's separate repository